src/site/content/en/fast/reduce-javascript-payloads-with-code-splitting/index.md

```diff
@@ -23,6 +25,18 @@ significantly. Instead of shipping all the JavaScript to your user as soon as
 the first page of your application is loaded, split your bundle into
 multiple pieces and only send what's necessary at the very beginning.
 
+## Why is code splitting beneficial?
+
+Code splitting is a technique that seeks to minimize startup time. When we ship less JavaScript at startup, we can get applications to be [interactive faster](/tti/) by minimizing main thread work during this critical period.
+
+When it comes to [Core Web Vitals](/vitals/), reducing JavaScript payloads downloaded at startup will contribute to better [First Input Delay (FID)](/fid/) and [Interaction to Next Paint (INP)](/inp/) times. The reasoning behind this is that, by freeing up the main thread, the application is able to respond to user inputs more quickly by reducing JavaScript parse, compile, and execution-related startup costs.
+
+Depending on your website's architecture&mdash;particularly if your website relies heavily on client-side rendering&mdash;reducing the size of JavaScript payloads responsible for rendering markup may lead to improved [Largest Contentful Paint (LCP)](/lcp/) times in some situations. This can occur when there are resources that are [delayed in being discovered by the browser](/optimize-lcp/#optimize-when-the-resource-is-discovered) until after client-side markup is completed, or when the main thread is too busy to [render that LCP element](/optimize-lcp/#2-eliminate-element-render-delay). Both scenarios can delay the LCP time for the page.
```

Comment:
Actually two more nits, and then I'll swear I'll leave you alone:

- "may" and "in some situations" are basically the same.
- should be clear the LCP resource is the delay we're concerned about.

```suggestion
Depending on your website's architecture&mdash;particularly if your website relies heavily on client-side rendering&mdash;reducing the size of JavaScript payloads responsible for rendering markup may lead to improved [Largest Contentful Paint (LCP)](/lcp/) times. This can occur when the LCP resource is [delayed in being discovered by the browser](/optimize-lcp/#optimize-when-the-resource-is-discovered) until after client-side markup is completed, or when the main thread is too busy to [render that LCP element](/optimize-lcp/#2-eliminate-element-render-delay). Both scenarios can delay the LCP time for the page.
```